app/controllers/about_controller.rb

```diff
@@ -9,8 +9,6 @@ def index
 
     respond_to do |format|
       format.html do
-        # @list = list
-        # store_preloaded(list.preload_key, MultiJson.dump(TopicListSerializer.new(list, scope: guardian)))
```

Comment:
Hmm not sure why it was commented out. @nlalonde Do you happen to recall why? I checked the commit for this and I didn't find anything.